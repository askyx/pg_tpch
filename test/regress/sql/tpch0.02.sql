select tpch_cleanup();
select * from dbgen(0.02);
-- select * from tpch;
-- select * from tpch;
-- select * from tpch(1,1,1);
-- select * from tpch(2,2,2);
