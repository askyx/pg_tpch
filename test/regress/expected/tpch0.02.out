select tpch_cleanup();
 tpch_cleanup 
--------------
 t
(1 row)

select * from dbgen(0.02);
   tab    | row_count 
----------+-----------
 region   |         5
 nation   |        25
 supplier |       200
 customer |      3000
 part     |      4000
 partsupp |     16000
 orders   |     30000
 lineitem |    120515
(8 rows)

-- select * from tpch;
-- select * from tpch;
-- select * from tpch(1,1,1);
-- select * from tpch(2,2,2);
